ui/app/mainui/AppMain.qml

```diff
@@ -1333,6 +1333,7 @@ Item {
             property var selectedAccount
             property bool isBridgeTx
             sourceComponent: SendModal {
+                onlyAssets: false
```

Comment:
left over?

Reply:
not really, most instances of the SendModal we have in the app are meant to send ERC20 tokens, except for this one where sending of NFTs is enabled.